src/libasr/pass/intrinsic_functions.h

```diff
@@ -924,6 +925,38 @@ namespace Shiftr {
 
 } // namespace Shiftr
 
+namespace Rshift {
+
+    static ASR::expr_t *eval_Rshift(Allocator &al, const Location &loc,
+            ASR::ttype_t* t1, Vec<ASR::expr_t*> &args) {
+        int64_t val1 = ASR::down_cast<ASR::IntegerConstant_t>(args[0])->m_n;
+        int64_t val2 = ASR::down_cast<ASR::IntegerConstant_t>(args[1])->m_n;
+        int64_t val = val1 >> val2;
+        return make_ConstantWithType(make_IntegerConstant_t, val, t1, loc);
+    }
+
+    static inline ASR::expr_t* instantiate_Rshift(Allocator &al, const Location &loc,
+            SymbolTable *scope, Vec<ASR::ttype_t*>& arg_types, ASR::ttype_t *return_type,
+            Vec<ASR::call_arg_t>& new_args, int64_t /*overload_id*/) {
+        declare_basic_variables("_lcompilers_rshift_" + type_to_str_python(arg_types[0]));
```

Comment:
This one is not in the runtime.

Reply:
If it is not used, then I think we can just use an empty string:
```suggestion
        declare_basic_variables("");
```
That way there is no confusion that it calls into the runtime library.